pkg/importer/dataStream_test.go

```diff
@@ -25,6 +46,78 @@ func createDataStream(ep, accKey, secKey string) *dataStream {
 	return ds
 }
 
+func createDataStreamMulti(testfile, ep string) *dataStream {
+	// set our directory for images
+	imageDir, _ := filepath.Abs("../../test/images")
```

Comment:
how do you know where the CWD is so that you're assured you need to hop up 2 dir levels?

Reply:
CWD is set within processes of the `go test` tool.  If `go test ./...` is run, it `cd`'s to the test's dir.  Otherwise, you must `cd` to the test dir manually, then exec `go test`  In either case, the relative path will be correct.

Reply:
ok, thanks. A comment describing this dependency on `go test ...` would be nice.

Reply:
ack

Reply:
I'm not sure CDI is the place to be explaining the workings of the `go` tool.